style/ruby/.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+AllCops:
+  Exclude:
+    - 'Gemfile'
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'spec/spec_helper.rb'
```

Comment:
Why exclude `Gemfile`, `config` and `spec_helper`? They seem easy enough to tidy (with or without the automation..) and it might catch some problems.. or, it did for Spotlight.